### PR TITLE
Add radius2john python script

### PR DESCRIPTION
**Pull request by MaximeGoyette**
Hi,

I found myself in a situation where I needed to use the radius2john perl script, but had some trouble making it work (mostly because I have no knowledge of perl whatsoever). 
So I wrote a new script in python to achieve my goal and I wanted to share it with you guys.

I'm open to suggestion on whether or not I need to modify my code.

Thanks!

### CODE REVIEW

File: run/radius2john.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# This software is Copyright (c) 2019 Maxime GOYETTE <maxgoyette0-at-gmail.com>,
+# and it is hereby released to the general public under the following terms:
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted.
+#
+# Utility to bruteforce RADIUS shared-secret
+# Usage: ./radius2john.py <pcap files>
+#
+# This script depends on Scapy (https://scapy.net)
+# To install: pip3 install scapy
```
Comment (kholia) [inline run/radius2john.py]: `pip3 install --user scapy` is recommended.

File: run/radius2john.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+            file_path = os.path.abspath(filename)
+
+            print('{ip}({file_path}):$dynamic_{type}${hash}$HEX${salt}'.format(
+                file_path=file_path,
+                ip=ips[1],
+                type=hash_type,
+                hash=hash_content,
+                salt=salt
+            ))
```
Comment (kholia) [inline run/radius2john.py]: Newline missing at the end of the file.

**Comment by kholia** [comment]
Please squash all your commits into a single one.

**Comment by kholia** [comment]
@MaximeGoyette Can you please share some test .pcap files for validating this PR?

**Comment by MaximeGoyette** [comment]
Hi!

I have a couple of samples to validate.
[radius-samples.zip](https://github.com/magnumripper/JohnTheRipper/files/2810182/radius-samples.zip)

I'm still figuring out how to squash commits :smile:

**Comment by MaximeGoyette** [comment]
Can't we use the built-in "[squash and merge](https://github.blog/2016-04-01-squash-your-commits/)" option in Github?

**Comment by kholia** [comment]
> Can't we use the built-in "squash and merge" option in GitHub?

It has its own problems (besides forgetting to use it at the last moment before merge).

**Pull request merged**